sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -65,25 +62,37 @@ public DefaultPodSpec(
             @JsonProperty("task-specs") List<TaskSpec> tasks,
             @JsonProperty("placement-rule") PlacementRule placementRule,
             @JsonProperty("volumes") Collection<VolumeSpec> volumes,
+            @JsonProperty("pre-reserved-role") String preReservedRole,
```

Comment:
`prereserved-role`? `reserved-role`? having a dash between pre and reserved makes it look like pre is a word

(here and several cases below)